src/js/components/PlacementConstraintsFrameworkAdapter.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import Batch from "#SRC/js/structs/Batch";
+import PlacementConstraintsPartial
+  from "#SRC/js/components/PlacementConstraintsPartial";
+import BatchContainer from "#SRC/js/components/BatchContainer";
+import {
+  JSONReducer,
+  JSONParser
+} from "#PLUGINS/services/src/js/reducers/serviceForm/JSONReducers/Constraints";
+import { combineReducers } from "#SRC/js/utils/ReducerUtil";
+import {
+  FormReducer
+} from "#PLUGINS/services/src/js/reducers/serviceForm/FormReducers/Constraints";
+import CreateServiceModalFormUtil
+  from "#PLUGINS/services/src/js/utils/CreateServiceModalFormUtil";
+
+const jsonReducer = combineReducers({ constraints: JSONReducer });
```

Comment:
Ah interesting, this is something I was missing when trying to make this work! I need to look more into what the point of this is.

Reply:
Yes, `combineReducers` not only combines them, but also injects the state into the resulting reducer, referring to your comment on `this` wasn't being available.